### PR TITLE
show erc list in editor

### PR DESCRIPTION
**Pull request by Aniket-Engg**
Show a list of listed erc interfaces and standards when user types `erc` in the editor.

On clicking enter, relevant code should be pasted in the code according to selection.

Inspired from: https://github.com/juanfranblanco/vscode-solidity/tree/master#erc-erc-drafts-and-smart-contracts-snippets--reference

erc details are fetched from: https://github.com/juanfranblanco/vscode-solidity/blob/master/snippets/solidity.json

### CODE REVIEW

File: libs/remix-ui/editor/src/lib/providers/completion/completionGlobals.ts [hunk 1]
```diff
@@ -188,7 +188,145 @@ export function getCompletionSnippets(range: monacoTypes.IRange, monaco): monaco
       insertText: '// SPDX-License-Identifier: MIT',
       insertTextRules: monaco.languages.CompletionItemInsertTextRule.InsertAsSnippet,
       range
+    },
+    /* eslint-disable */
+    // Related to https://github.com/juanfranblanco/vscode-solidity/blob/master/snippets/solidity.json
+    {
+      label: 'erc20i',
```
Comment (yann300) [inline libs/remix-ui/editor/src/lib/providers/completion/completionGlobals.ts:195]: probably the labels should be more explicit, like `generate erc20i` for instance.

Comment (Aniket-Engg) [reply libs/remix-ui/editor/src/lib/providers/completion/completionGlobals.ts:195]: > probably the labels should be more explicit, like `generate erc20i` for instance.

that is a keyword to show the list

Comment (Aniket-Engg) [reply libs/remix-ui/editor/src/lib/providers/completion/completionGlobals.ts:195]: > probably the labels should be more explicit, like `generate erc20i` for instance.

that is a keyword to show the list

Comment (Aniket-Engg) [reply libs/remix-ui/editor/src/lib/providers/completion/completionGlobals.ts:195]: > btw just wondering, do we already have snippet for generating loops?

yes, we have

Comment (yann300) [reply libs/remix-ui/editor/src/lib/providers/completion/completionGlobals.ts:195]: > that is a keyword to show the list. We can update the detail field

yes, in  https://github.com/juanfranblanco/vscode-solidity/raw/master/screenshots/ercautocomplete1.png
there's a quick and a full description associated with the keyword.

Comment (Aniket-Engg) [reply libs/remix-ui/editor/src/lib/providers/completion/completionGlobals.ts:195]: > > probably the labels should be more explicit, like `generate erc20i` for instance.
> 
> that is a keyword to show the list. We can update the detail field

updated @yann300

**Comment by yann300** [comment]
ok I see thanks for the update, is this also possible to add the full description, as in the screenshot above?

**Comment by Aniket-Engg** [comment]
> ok I see thanks for the update, is this also possible to add the full description, as in the screenshot above?

it is there, when you click on `>` of particular option

**Review by bunsenstraat — APPROVED** [review]

**Pull request merged**